### PR TITLE
move static data to separate files

### LINKED ISSUE #149: Move Data to Separate File
- Currently lots of components rely on static data to map through components and what not
- Lets move that data to a separate file, this way the component file will only contain code regarding the rendering of the component and the actual content will come from another file

### PR DESCRIPTION
**Pull request by pondohoo**
named the files to where they'll appear next to the component they export to; lmk if there is a better convention

### CODE REVIEW

**Review by shahdivyank — CHANGES_REQUESTED** [review]
can we move all the data files into a folder called `data` just to prevent confusion with components

File: components/data/CustomToolbarFilters.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export const CustomToolbarFilters = [
```
Comment (shahdivyank) [inline components/data/CustomToolbarFilters.js]: rename to Filters

File: components/data/FooterSocialLinks.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import {
+	FaDiscord,
+	FaInstagram,
+	FaSlack,
+	FaYoutube,
+	FaMediumM,
+	FaGithub,
+} from "react-icons/fa";
+
+export const FooterSocialLinks = [
```
Comment (shahdivyank) [inline components/data/FooterSocialLinks.js]: Rename to Socials

File: components/data/OfficersArray.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
+export const OfficersArray = [
```
Comment (shahdivyank) [inline components/data/OfficersArray.js]: Rename to Officers

File: components/data/OfficersColors.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+export const OfficersColors = [
```
Comment (shahdivyank) [inline components/data/OfficersColors.js]: Rename to Colors

**Review by shahdivyank — APPROVED** [review]

**Review by skyguy-netizen — APPROVED** [review]

**Pull request merged**